src/chainlit/frontend/src/components/organisms/chat/index.tsx

```diff
@@ -143,6 +158,16 @@ const Chat = () => {
             />
           </ErrorBoundary>
         )}
+        {!messages.length && !exemptHostNames.includes(window.location.href) ? (
+          <>
+            <Iframe
```

Comment:
In terms of styling, can/should we remove the frame border on this? Also, I'm wondering if you considered embedding the iframe in an [Alert](https://mui.com/material-ui/react-alert/) component of some kind since that might make it stand out a bit.

Reply:
Good point! An info alert could work for in case.

<img width="1897" alt="Screenshot 2024-07-09 at 4 04 50 PM" src="https://github.com/Harvard-University-iCommons/chainlit/assets/29421667/ced83ec7-d5ed-47f9-ad01-aa5adab6d97f">

Reply:
Seems like dark mode does not like Iframe text color change

<img width="1769" alt="Screenshot 2024-07-09 at 4 34 39 PM" src="https://github.com/Harvard-University-iCommons/chainlit/assets/29421667/95313f7a-19f1-43d1-a5a3-9d2781ae4786">

Reply:
Since we can't access the CSS styles within the Iframe due to `same-origin-policy` I am opting to create two different `html` versions(dark and light themes) with different URLs.
Note: links in dark theme should turn yellow as per the dark styling.
<img width="1719" alt="Screenshot 2024-07-10 at 2 48 16 PM" src="https://github.com/Harvard-University-iCommons/chainlit/assets/29421667/ea907451-5065-496a-af30-63e8f1b775ca">